### DISCUSSION TITLE
Release date for version 4.2.6

### DISCUSSION THREAD

**Discussion by TPIvan** [havit/Havit.Blazor#651] [Q&A] [answered]
I have been following the development of version 4.2.6 and I noticed that there have been six prereleases so far. I am wondering if there is a timeline for the final release of this version. Thank you for your hard work and dedication to this project.

**Comment by hakenr** [comment, marked as answer]
Since we have not received any feedback, either external or internal, indicating that the new changes have caused any issues, I plan to release the new version today.
